garlic_functions.py

```diff
@@ -274,3 +274,14 @@ def humanize_text(message: Message, text: str) -> str:
         for role in message.role_mentions:
             text = text.replace(role.mention, role.name)
         return text
```

Comment:
This should be removed. Why are there two versions of humanize_text, and why is one indented like this?